hawc/apps/summary/migrations/0052_visual_image_alter_visual_visual_type.py

```diff
@@ -33,6 +33,7 @@ class Migration(migrations.Migration):
                     (6, "exploratory heatmap"),
                     (7, "plotly"),
                     (8, "static image"),
+                    (9, "PRISMA"),
```

Comment:
I had to update a migration since the database choices were altered; I didn't have to create one since this doesn't change the database